src/resources/roboto.js

```diff
@@ -1,10 +1,8 @@
 export {}; // for Babel to treat as a module
 
-const documentContainer = document.createElement("template");
-documentContainer.setAttribute("style", "display: none;");
+const styleContainer = document.createElement("style");
 
-documentContainer.innerHTML = `<style>
-@font-face {
+styleContainer.innerHTML = `@font-face {
```

Comment:
@KTibow We have similar pattern for `ha-style.ts` so we should do the same for this one too if there is no objection for this. 😅 So wait bram feedback

Reply:
My explanation:
Some Polymer components were converted to JS. They used template tags.
In this case, we can use `<style>` because it's a simple style. I'm not sure if it would work in `ha-style` because we use `<custom-style>` there - maybe it would though? I guess I'll wait for Bram too.

Reply:
I think we can just use a style here indeed. I did try/check replacing  `custom-style` ones, but that was harder than I though...